### PR TITLE
Add documentation for KubeOps operation "delete"

### PR DESCRIPTION
**Pull request by muffl0n**
## Change Overview

KubeOps was introduced in #1102 but documentation for operation "delete" is missing from "Kanister Functions". 

## Pull request type

Please check the type of change your PR introduces:
- [ ] :construction: Work in Progress
- [ ] :rainbow: Refactoring (no functional changes, no api changes)
- [ ] :hamster: Trivial/Minor
- [ ] :bug: Bugfix
- [ ] :sunflower: Feature
- [x] :world_map: Documentation
- [ ] :robot: Test

## Issues

- #1102

## Test Plan

- [x] :muscle: Manual
- [ ] :zap: Unit test
- [ ] :green_heart: E2E

### CODE REVIEW

**Comment by muffl0n** [comment]
Resulting docs:  Bildschirmfoto 2022-04-07 um 16 40 31

**Comment by ihcsim** [comment]
This LGTM :+1:. While you are here :wink:, can you also help us to add the `Outputs` subsection to describe the function outputs? Looking at the code, these should be the relevant fields :point_right:  https://github.com/kanisterio/kanister/blob/c71b5aa161e589364a40d5514d03e7052af0030f/pkg/apis/cr/v1alpha1/types.go#L58-L75. Thanks!

**Comment by muffl0n** [comment]
As I'm on maternity leave at the moment, I don't have the time. But this PR can surely be merged without the outputs. Documentation for this function might not be perfect, but a step in the right direction.

File: docs/functions.rst [hunk 1]
```diff
@@ -1221,7 +1221,16 @@ Example:
...
+    args:
+      operation: delete
+      objectReference:
+        apiVersion: "{{ .Phases.createDeploy.Output.apiVersion }}"
+        group: "{{ .Phases.createDeploy.Output.group }}"
+        resource: "{{ .Phases.createDeploy.Output.resource }}"
+        name: "{{ .Phases.createDeploy.Output.name }}"
+        namespace: "{{ .Phases.createDeploy.Output.namespace }}"
 
 Wait
```
Comment (pavannd1) [inline docs/functions.rst:1235]:
```suggestion

Wait
```
extra line required for better readability

**Review by ihcsim — APPROVED** [review]

**Pull request merged**